docs/pages/api-reference/liveblocks-node.mdx

```diff
@@ -324,15 +1000,243 @@ returns the same response.
 
 ```ts
 const comment = await liveblocks.getComment({
-  roomId: "my-room",
-  threadId: "my-thread",
-  commentId: "my-comment",
+  roomId: "my-room-id",
+  threadId: "my-thread-id",
+  commentId: "my-comment-id",
 });
 ```
 
 ---
 
-## WebhookHandler [#WebhookHandler]
+## Error handling [#Error-handling]
+
+Errors in our API methods, such as network failures, invalid arguments, or
```

Comment:
@marcbouchenoire Added a section for error handling. Do you mind taking a look when are free?

Reply:
Nice! Great addition, I made a few tiny tweaks and I think it's ready to go.